frontend/services/entities/teams.ts

```diff
@@ -27,6 +35,12 @@ export default {
 
     return sendRequest("DELETE", path);
   },
+  load: (teamId: number) => {
```

Comment:
What do you think about typing the return for services methods? Let me know what you think of `/services/entities/software` as an example of typing the returned promise?

Also curious what you think about the idea of typing the request and response interfaces in the services file rather than the entity interface as a way to keep the API typings closer to the the underlying concern?

Reply:
Sounds good. I'll write it up as an issue.